setup.sh

```diff
@@ -226,6 +232,7 @@ function usage()
   say "--uninstall | -u     Uninstall $app_name"
   say '--skip-checks        Skip checks (use this when packaging)'
   say '--skip-docs          Skip creation of man pages (use this when installing)'
+  say '--system    | -s     Install kw system wide (use this when installing)'
```

Comment:
You also need to manage the uninstall in this scenario.

Reply:
Yes, we managed uninstall too. I left some comments about it in the code:

```bash
# if system wide, change the installation directories.
# This approach works both for installing and uninstalling KW.
if [[ "${SYSTEMWIDE}" == 1 ]]; then
  use_system_wide_installation_directories
fi
```